0.19.3 - ReactNativeBlobUtilPackage.java:30: error: cannot find symbol BuildConfig

Hi, after upgrading to 0.19.3 I'am not able to run my android app from the command line.

**I have this error.**

..../node_modules/react-native-blob-util/android/src/main/java/com/ReactNativeBlobUtil/ReactNativeBlobUtilPackage.java:30: error: cannot find symbol
            boolean isTurboModule = BuildConfig.IS_NEW_ARCHITECTURE_ENABLED;
                                    ^
  symbol:   variable BuildConfig
  location: class ReactNativeBlobUtilPackage

**This is my current environment:**

```
System:
  OS: macOS 14.1.1
  CPU: (12) arm64 Apple M2 Max
  Memory: 12.25 GB / 64.00 GB
  Shell:
    version: "5.9"
    path: /bin/zsh
Binaries:
  Node:
    version: 21.2.0
    path: /var/folders/pv/51qh3b417f995x35c1dfpyd80000gn/T/yarn--1700178466961-0.7102813556842231/node
  Yarn:
    version: 1.22.21
    path: /var/folders/pv/51qh3b417f995x35c1dfpyd80000gn/T/yarn--1700178466961-0.7102813556842231/yarn
  npm:
    version: 10.2.3
    path: /opt/homebrew/bin/npm
  Watchman: Not Found
Managers:
  CocoaPods:
    version: 1.13.0
    path: /Users/xxxxxx/.rvm/rubies/ruby-3.1.4/bin/pod
SDKs:
  iOS SDK:
    Platforms:
      - DriverKit 23.0
      - iOS 17.0
      - macOS 14.0
      - tvOS 17.0
      - watchOS 10.0
  Android SDK: Not Found
IDEs:
  Android Studio: 2022.3 AI-223.8836.35.2231.11005911
  Xcode:
    version: 15.0.1/15A507
    path: /usr/bin/xcodebuild
Languages:
  Java:
    version: 17.0.9
    path: /usr/bin/javac
  Ruby:
    version: 3.1.4
    path: /Users/xxxxxxxxxxx/.rvm/rubies/ruby-3.1.4/bin/ruby
npmPackages:
  "@react-native-community/cli": Not Found
  react:
    installed: 18.2.0
    wanted: 18.2.0
  react-native:
    installed: 0.72.7
    wanted: ^0.72.4
  react-native-macos: Not Found
npmGlobalPackages:
  "*react-native*": Not Found
Android:
  hermesEnabled: true
  newArchEnabled: false
iOS:
  hermesEnabled: true
  newArchEnabled: false

```

Thanks

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -46,7 +46,7 @@ if (isNewArchitectureEnabled()) {
 android {
     def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
     if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
-        namespace "com.ronradtke.rnblobutil"
+        namespace "com.ReactNativeBlobUtil"
     }
     compileSdkVersion safeExtGet('compileSdkVersion', 30)
     buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')

--- a/android/build.gradle
+++ b/android/build.gradle
@@ -47,6 +47,9 @@ android {
     def agpVersion = com.android.Version.ANDROID_GRADLE_PLUGIN_VERSION
     if (agpVersion.tokenize('.')[0].toInteger() >= 7) {
         namespace "com.ReactNativeBlobUtil"
+        buildFeatures {
+            buildConfig true
+        }
     }
     compileSdkVersion safeExtGet('compileSdkVersion', 30)
     buildToolsVersion safeExtGet('buildToolsVersion', '28.0.3')

